src/app/schools/schools-list/schools-list.component.html

```diff
@@ -35,7 +35,7 @@ <h1>Schools List</h1>
 </mat-expansion-panel>
 
 <div *ngIf="schoolDataSource">
-  <table mat-table [dataSource]="schoolDataSource" matSort class="mat-elevation-z8 table-list" width='100%'>
+  <mat-table mat-table [dataSource]="schoolDataSource" matSort class="mat-elevation-z8 table-list" width='100%'>
```

Comment:
Angular Material documentation recommends to use `<table mat-table>` (https://material.angular.io/components/table). I think that has some advantages over `<mat-table>` regarding how browsers handle it. Any particular reason to change it here?